setup_database.sh

```
@@ -0,0 +1,439 @@
+echo "============================================"
+echo "  ArXiv Repository Database Setup"
+echo "  Production-grade with User Management"
+echo "============================================"
+echo
+
+# Check for root password
+if [ -z "$ROOT_PASSWORD" ]; then
+    # Try to load from .env file
+    if [ -f ".env" ]; then
+        print_info "Loading credentials from .env file..."
+        export $(grep -v '^#' .env | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)